### PR TITLE
while hallucinating, chance your coworkers may look like a hostile mob

### PR DESCRIPTION
**Pull request by MadmanMartian**

### CODE REVIEW

File: code/modules/flufftext/Hallucination.dm [hunk 1]
```diff
@@ -309,7 +309,33 @@ mob/living/carbon/proc/handle_hallucinations()
...
+
+				var/list/mob_list=list()
+				for(var/mob/living/M in viewers(src))
+					mob_list |= M
+
+				if(mob_list.len)
+					L = pick(mob_list)
+
+				var/mob/living/simple_animal/random_mob = pick(existing_typesof(/mob/living/simple_animal/hostile))
+				var/image/horror = image(initial(random_mob.icon), initial(random_mob.icon_state))
```
Comment (unid15) [inline code/modules/flufftext/Hallucination.dm]: Copy appearance by doing ``horror.appearance = initial(random_mob.appearance)`` instead

**Review by DamianX — CHANGES_REQUESTED** [review]
See unid's recommendation

**Comment by Killette2** [comment]
(kills coworker on purpose knowing about this feature)
hehe oops I was hallucinating! my character is crazy!
Seems like an administration nightmare

**Comment by Pathid** [comment]
That's just good roleplaying. 

Jokes aside, this feature could definitely use logging for when and who is being disguised. Otherwise you get a situation where you can never act on it on that 1/100 chance it does happen because there's no reason for the admins to believe you're not just a shitter.

**Comment by Killette2** [comment]
What if it is logged that you are hallucinating, that still doesn't solve the following issue. The player can chose to attack the person they're hallucinating, fully knowing that it's a hallucination but doing it because the window for valids is open. Sure it's "roleplay" but it's a shitty reason to do serious possible harm to someone.

**Pull request merged**